context/app/static/js/App.jsx

```diff
@@ -40,3 +41,14 @@ export default function App(props) {
     </ThemeProvider>
   );
 }
+
+App.propTypes = {
+  flaskData: PropTypes.exact({
+    entity: PropTypes.object,
+    flashed_messages: PropTypes.array,
+    provenance: PropTypes.object,
+    vitessce_conf: PropTypes.object,
+  }).isRequired,
+};
+
+export default App;
```

Comment:
Is this a better practice than exporting at the definition?

Reply:
More of a preference than a better practice. Not strongly opinionated on this one so if you believe I should revert the change where possible, I can.

Reply:
No, I don't care one way or the other, just curious. Just leave it. Thanks!